### PR TITLE
Copy local file to pod using tar

### PR DESCRIPTION
**Pull request by lazarkrstic**

### CODE REVIEW

File: examples/cp/demo.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
```
Comment (tg123) [inline examples/cp/demo.txt:1]: could you please remove this file?

**Review by tg123 — COMMENTED** [review]
thanks

I will make inner websocket chunked

**Review by tg123 — APPROVED** [review]
/LGTM

**Comment by lazarkrstic** [comment]
Can you remove label `invalid-commit-message` , I amended my privious git message

**Comment by tg123** [comment]
do not put `#` in message, just remove it and push -f

**Review by tg123 — APPROVED** [review]
/LGTM

**Comment by cody-malcolm** [comment]
@lazarkrstic Thanks for doing this. When I use this example, the file is copied to the pod successfully, but then the thread blocks on the error checking starting at line 96...specifically being blocked by `streamReader.EndOfStream`. I have tried various other ways to perform the same check (such as ReadLine, Read, or stdError.Length) and they all block. If I remove the error check entirely, it will block in Kubernetes.Exec.cs at `await errorReader.ReadToEndAsync().ConfigureAwait(false);`. Do you know why this might be happening or what can be done to fix it? I'm out of ideas to fix it. Thanks in advance.

**Comment by tg123** [comment]
@cody-malcolm what is your base image?

only ubuntu' tar passed the test


see comment here
https://github.com/kubernetes-client/csharp/blob/fd3931c994f6ce6fab60253acf72edb7d750ff30/tests/E2E.Tests/MinikubeTests.cs#L654

**Comment by cody-malcolm** [comment]
@tg123 We are using node:16.15.0-alpine base image. Based on Alpine Linux.

**Pull request merged**